src/v/test_utils/test.h

```diff
@@ -173,4 +175,7 @@ class seastar_test : public ::testing::Test {
 #define ASSERT_LT_CORO(val1, val2) GTEST_ASSERT_LT_CORO(val1, val2)
 #define ASSERT_LE_CORO(val1, val2) GTEST_ASSERT_LE_CORO(val1, val2)
 #define ASSERT_NE_CORO(val1, val2) GTEST_ASSERT_NE_CORO(val1, val2)
+
+#define GTEST_SKIP_CORO() GTEST_SKIP_CORO_("")
+
```

Comment:
This macro appears to be unused in this PR. It also should have a test in `test_utils/tests/gtest_tests.cc`.